recipe/meta.yaml

```diff
@@ -1,65 +1,56 @@
 {% set name = "mizani" %}
-{% set version = "0.9.2" %}
-{% set bundle = "tar.gz" %}
-{% set hash = "1d481a4dc673caa9b7cfdc6505b9401f0e9a9f43434d748df0678a1a4017b0e2" %}
-{% set build = 0 %}
+{% set version = "0.11.3" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.{{ bundle }}
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ bundle }}
-  sha256: {{ hash }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a8e0b16627442bda2899e39f834ff004eb49f2da874f01f13aa329d073d6adc5
 
 build:
-  number: {{ build }}
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: True  # [py<39]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   host:
-    - python >=3.8.0
+    - python
     - pip
     - setuptools
     - setuptools_scm
-
+    - wheel
   run:
-    - python >=3.8.0
-    - numpy >=1.19.0
-    - scipy >=1.5.0
-    - pandas >=1.3.5
-    - matplotlib-base >=3.5.0
-    - backports.zoneinfo  # py<3.9 conda does not properly support environment markers
-    - tzdata  # os == windows
+    - python
+    - numpy >=1.23.0
+    - scipy >=1.7.0
+    - pandas >=2.1.0
+    - tzdata  # [win]
 
 test:
+  imports:
+    - mizani
   requires:
     - pip
     - pytest
     - pytest-cov
-  imports:
-    - mizani
-    - mizani.external
-    - mizani.tests
   source_files:
-    - '*'
+    - "tests"
+    - pyproject.toml
   commands:
-    # The pip version can reported incorrectly if setuptools_scm isn't available
-    # at build time
-    - pip list | grep -i {{ name }} | grep {{ version }}
-    - pip check
-    - echo $PWD
-    - ls -larth
-    - pytest
+    - pip check  # [not win]
+    - pytest -vv
```

Comment:
The test failed on `win64` for **py311**:

```
>           raise ValueError(

                "Cannot get a null value for type: {}".format(type(x[0]))

E           ValueError: Cannot get a null value for type: <class 'numpy.int32'>

..\_test_env\Lib\site-packages\mizani\utils.py:387: ValueError
...

FAILED tests/test_bounds.py::test_censor - ValueError: Cannot get a null value
```

Is this an upstream issue or other `numpy` incompatibility?